CHANGELOG.rst

```diff
@@ -1,6 +1,33 @@
 Changelog
 =========
 
+0.15.7 (2020-06-xx)
+-------------------
+
+- [NEW] Added a number of built-in format strings. See the `docs <https://arrow.readthedocs.io/#built-in-formats>`_ for a complete list of supported formats. For example:
+
+.. code-block:: python
+
+    >>> arw = arrow.utcnow()
+    >>> arw.format(arrow.FORMAT_COOKIE)
+    'Wednesday, 27-May-2020 10:30:35 UTC'
+
+- [NEW] Added full Python 3.9 compatibility.
```

Comment:
Maybe a mention for pypy3 also which we added to the github actions since the last release?